index.html

```diff
@@ -799,8 +799,8 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
           <!-- FIXME: Export capabilities, constraints and settings so that we can use them here. -->
           <p>Update <var>track</var>'s capabilities, constraints and settings according <var>track</var>'s underlying source.</p>
         </li>
-        <li>
-          <p>[=Fire an event=] named <var>configurationchange</var> on <var>track</var>.</p>
+        <li><p>[=Queue a task=] on [=current settings object=]'s [=environment settings object/responsible event loop=] to
+          [=fire an event=] named <var>configurationchange</var> on <var>track</var>.</p>
```

Comment:
We probably want to queue a task that would both update the track slots and fire the event synchronously, that would mean the task would run step 2, 3 and 4 of the current algorithm.

Reply:
Fixed. Thanks!